SubmissionTest.java

```diff
@@ -603,6 +603,272 @@ public void testGetCostumeCountForSpriteValid()
     }
     
     /**
+     * Test getControlBlocksForStage method.
+     */
+    @Test
+    public void testGetControlBlocksForStage()
+    {
+        setUpSubmission(submissions[1]);
+        submissions[1].countBlockCategoriesForStage();
+
+        int expected = 1;
+        int actual = submissions[1].getControlBlocksForStage();
+        assertEquals("should be equal", expected, actual);
```

Comment:
I think you're right!  It does make clear which is the actual and expected parameter without knowledge of the method definition .  A-okay with me!